terraform/datadog/aws.tf

```diff
@@ -0,0 +1,17 @@
+resource "datadog_monitor" "ec2-cpu-credits" {
+  name               = "${format("%s EC2 CPU credits", var.env)}"
+  type               = "query alert"
+  message            = "Instance is low on CPU credits and may start to perform badly. See: https://government-paas-team-manual.readthedocs.io/en/latest/support/responding_to_alerts/#cpu-credits"
+  escalation_message = "Instance is out of CPU credits and may perform badly. See: https://government-paas-team-manual.readthedocs.io/en/latest/support/responding_to_alerts/#cpu-credits"
```

Comment:
As far as I understand, this escalation message is only used when datadog renotifies, but we don't enable renotification.
Could we just remove it?